imports/ui/styles/_cards.scss

```diff
@@ -0,0 +1,5 @@
+small {
```

Comment:
I think this is way too small and unreadable from an accessibility standpoint

![image](https://user-images.githubusercontent.com/3891869/55602247-87768080-5719-11e9-9c55-9bd7edc57aa9.png)

in comparison to using some built in bootstrap markup

![image](https://user-images.githubusercontent.com/3891869/55602432-66faf600-571a-11e9-930f-60671634e239.png)

```html
<div class="card-footer text-right">
    <small class="text-muted">(GMT-12:00) International Date Line West</small>
</div>
```

Reply:
What about 11px?

This was what I had in my HTML:
      ```  <Card.Footer><small className="text-muted">{ timezone }</small></Card.Footer>```

But the default caused the timezone to be on two lines:
![image](https://user-images.githubusercontent.com/4512699/55602941-fbfeee80-571c-11e9-9af7-df2fc3d6d297.png)

Maybe we can decrease the line height here?

Reply:
Put this into a separate issue: https://github.com/codebuddies/cb-connect/issues/48